DigitalExcellence/dex-frontend
# stripped tags on highlighted projects

## Description

I've created a custom pipe to strip HTML tags from a string. This is applied on the top highlighted projects component.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to home page (make sure project is highlighted)
2. Check to see if all HTML tags are stripped from the description

## Link to issue

Closes: #243
